src/opnsense/mvc/app/controllers/OPNsense/Proxy/forms/main.xml

```diff
@@ -223,23 +223,24 @@
             </field>
             <field>
                 <id>proxy.forward.sslbump</id>
-                <label>Enable SSL mode</label>
+                <label>Enable SSL inspection</label>
                 <type>checkbox</type>
                 <help><![CDATA[
-                  Enable sslbump mode,
-                  which makes the proxy act as a man in the middle between the internet and your clients.<br/>
+                  Enable SSL inspection mode,
+                  which allows to log HTTPS connections information, such as requested URL and/or
+                  make the proxy act as a man in the middle between the internet and your clients.<br/>
                   Be aware of the security implications before enabling this option.
-                  <br/><br/>
-                  Transparent HTTP proxy needs to be enabled and you need nat rules to reflect your traffic
-                  for this feature to work.<br/>
-                  <a href="/firewall_nat_edit.php?template=transparent_proxy&https=1"> Add a new firewall rule </a>
```

Comment:
@mmorev we have pluggable rules now, they could be used instead. though I liked the simplicity of the approach: rule for http transparent use, another one for https transparent use as it gives the flexibility to do the one or the other or both. plus the user could change the rule, set it for a specific range of sources or destinations. If we use a checkbox from the proxy, this flexibility will go away.